packages/curve-ui-kit/src/widgets/Footer/Sections.tsx

```diff
@@ -22,15 +22,15 @@ import { AppName } from '@ui-kit/shared/routes'
 
 function getDisclaimersTab(appName: AppName) {
```

Comment:
The name isn't very precise.. Maybe something like this is more clear?
```suggestion
function getDisclaimersQueryString(appName: AppName) {
```

Reply:
it's not used anymore, I removed it. Thanks